src/pages/cart_page/CartTotals.jsx

```
@@ -0,0 +1,49 @@
+          </label>
+          <input
+            type="text"
+            id="coupon"
+            placeholder="Coupon code"
+            className="border-1 px-2 py-1.5 text-sm grow border-gray-300 rounded-md"
+          />
+          <Button children="APPLY" />
+        </form>
+
+        <Link>
+          <Button children="PROCEED TO CHECKOUT" dynamicStyle="w-full" />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)